### ISSUE TITLE
Any news ? 

### ISSUE THREAD

**Issue by draco1544** [garet90/MotorMC#4]
Any news on this project ?

**Comment by 000exploit** [comment]
Unfortunately, looks like abandonware..

**Comment by garet90** [comment]
Yeah, I have stopped working on this project for the time being. I might come back to it when I have time. Anyone is free to continue it though

If I were to continue I might rewrite it to make different design choices lol
